.github/workflows/ExecuteQuery_build_and_test_on_main.yml

```diff
@@ -0,0 +1,28 @@
+name: ExecuteQuery_build_main
+
+on:
+  push:
+    branches:    
+      - main
+    paths:
+      - 'Frends.Salesforce.ExecuteQuery/**'
+  workflow_dispatch:
+    paths:
```

Comment:
Do we need paths with workflow dispatch?

Reply:
Not really.